guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -1,40 +1,71 @@
+You must have a {Team} subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
+All subscription information is available in your {Team} Customer Portal account.
 
-Use this chapter to import a Red{nbsp}Hat subscription manifest and manage the manifest within the {ProjectWebUI}.
+Use this chapter to import a {Team} subscription manifest and manage the manifest within the {ProjectWebUI}.
 
 .Subscription allocations and organizations
-
 You can manage more than one organization if you have more than one subscription allocation.
 {Project} requires a single allocation for each organization configured in {ProjectServer}.
-The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red Hat accounts.
+The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own {Team} accounts.
 
 .Future-dated subscriptions
-
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
-.Prerequisites
-* Ensure you have a Red{nbsp}Hat subscription manifest.
-ifdef::orcharhino[]
-You can create and download your manifest in the https://access.redhat.com/management/subscription_allocations[Red Hat Customer Portal].
-endif::[]
-ifndef::orcharhino[]
-** If your {Project} is connected, use the {RHCloud} to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
-endif::[]
+.Subscriptions service capabilities
+* Allows customers to view subscription usage.
+* When configured, it displays usage data from both connected and disconnected environments.
+* Tracks {Team} product usage and capacity at the fleet or account level.
+* Generates a daily snapshot of usage data across all tracked systems.
+* Displays data in a filterable dashboard at cloud.redhat.com.
+* Collects historical data for analytics, purchasing decisions, capacity planning, and risk mitigation.
+* Supports the transition to Simple Content Access (SCA), which enforces subscriptions across the organization rather than on individual systems.
```

Comment:
This just jumps right into a bulleted list without explaining what the Subscriptions service is or why I am reading this. Is the Subscriptions service a capability of Satellite? (no it isn't, but I wouldn't know that from reading this.)